examples/Cassie/simulator_drift.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "drake/multibody/rigid_body_tree.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "systems/framework/output_vector.h"
+
+class SimulatorDrift : public drake::systems::LeafSystem<double> {
```

Comment:
Some documentation needed. This looks like it adds the output of a random walk to the position vector, correct? This should be stated, and along with requirements on the size of the mean and covariance.